termwiz/src/input.rs

```diff
@@ -86,6 +86,12 @@ pub struct KeyEvent {
 #[cfg_attr(feature = "use_serde", derive(Serialize, Deserialize))]
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
 pub enum KeyCode {
+    // other keys when disabling its original use will show up as this keycode
+    // e.g. setxkbmap -option caps:none 
+    // prevents the key from operating as originally intended and
+    // makes it possible to utilize it as a leader/modifier with no oddities
+    VoidSymbol,
```

Comment:
I don't think there is a way to represent this for the terminal, so I'd be inclined to remove it from this particular enum and the corresponding matches in the termwiz/term crates

Reply:
It would seem my understanding of the code is still a bit green. I actually had wondered why are we defining a semi-duplicate `enum` of `KeyCodes`? I apparently did not look too deeply into the implementations of them and just naively plastered it everywhere I saw `CapsLock`.  **I will look into this further.**